wsmaster/che-core-api-account/src/main/java/org/eclipse/che/account/spi/jpa/JpaAccountDao.java

```diff
@@ -111,7 +111,9 @@ public void remove(String id) throws ServerException {
 
   @Transactional
   protected void doCreate(AccountImpl account) {
-    managerProvider.get().persist(account);
+    final EntityManager manager = managerProvider.get();
+    manager.persist(account);
+    manager.flush();
```

Comment:
Pls fix AccountDao.doRemove() in the same way.

Reply:
done